src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -89,6 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln('Installing assets as <comment>hard copies</comment>.');
         }
 
+        $validAssetDir = array();
```

Comment:
The variable name should be a plural. It is not a directory, but a list of directories.